src/Stratis.Bitcoin.Features.Miner/PosMinting.cs

```diff
@@ -243,14 +252,11 @@ public class CoinstakeContext
         /// <summary>Loop in which the node attempts to generate new POS blocks by staking coins from its wallet.</summary>
         private IAsyncLoop stakingLoop;
 
-        /// <summary>Indicates that the stake flag is not in progress.</summary>
-        public const int StakeNotInProgress = -1;
+        /// <summary>a flag that indicates if stake is on/off based on the <see cref="CurrentState"/> enum.</summary>
```

Comment:
Capital letter to start the sentence with

Reply:
Fixed